CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Our versioning strategy is as follows:
 * `[sitecore-jss-react]` `[sitecore-jss-nextjs]` FEaaS component will render 'staged' variant for editing and preview and 'published' variant for live site by default, unless variant is overriden via rendering parameters. ([#1433](https://github.com/Sitecore/jss/pull/1433))
 * `[templates/nextjs]` `[templates/angular]` `[templates/react]` `[templates/vue]` Pre-push hook for lint check ([#1427](https://github.com/Sitecore/jss/pull/1427))
 ([#1442](https://github.com/Sitecore/jss/pull/1442)) ([#1444](https://github.com/Sitecore/jss/pull/1444))
+* `[sitecore-jss-nextjs] Add a new handling for token $siteLang(context site language) in middleware redirect ([#1454](https://github.com/Sitecore/jss/pull/1454))
```

Comment:
Looks like a merge from dev was done to this branch. Makes it a bit difficult to sift through actual changes in this work item. In the future, I'd recommend merging dev to the integration branch first, then rebasing this one.

Reply:
Yeah, Illia has noted this too. This PR is a victim of an unnecessary dev merge, but it's too hard to revert that at this point.